src/lib/components/Cytoscape.react.js

```diff
@@ -266,10 +292,20 @@ class Cytoscape extends Component {
             sendSelectedEdgesData();
         });
 
-        cy.on('add remove', () => {
+        cy.on('add remove ', () => {
             refreshLayout();
         });
 
+        cy.on('position', 'node', (event) => {
+            if (typeof this.props.setProps === 'function') {
+                this.props.setProps({
+                    elements: this.props.elements.map((item) =>
```

Comment:
@Farkites Does this actually update the position of the elements? It looks like this would trigger a prop update with the new timestamp, but still using the old position of the elements.

But generally this feels cleaner -- so an approach like this is on the right track I think.

Looking into this a bit this AM.

Reply:
This approach works with: 

```
this.props.setProps({
    elements: cy.elements('').map((item) => {
         return {
             data: item.json().data,
             position: item.json().position,
         };
    }),
});
```

Although it introduces `position: {x: 0, y:0}` for edges...